epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -53,69 +55,82 @@
 	<body>
 		<section id="abstract">
 			<p>The <a href="https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32019L0882">European
-					Accessibility Act (EAA)</a> is an EU directive that establishes binding accessibility targets that
+					Accessibility Act (EAA)</a> is an EU directive that establishes binding accessibility requirements that
 				must be met by many different types of products and services to strengthen the rights of people with
 				disabilities to access goods and services, including ebooks, dedicated reading software, ereading
```

Comment:
This paragraph is a bit of a run-on sentence. You might want to break it in two at "services":

> ... products and services. It strengthens the rights of people ...